service/controller/resource/deployment/template/main.json

```diff
@@ -732,6 +732,7 @@
     {
       "apiVersion":"2016-09-01",
       "name":"virtual_network_setup",
+      "condition":"[equals(parameters('initialProvisioning'), 'Yes')]",
```

Comment:
We don't apply the virtual_network_setup deployment unless the three expected subnets are already in place. This means that if we want to change the name or cidr of the virtual network, we can't do it using the ARM deployment.